xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -740,11 +908,21 @@
         <Parameter Name="byteCount" Type="System.UInt32" />
       </Parameters>
       <Docs>
-        <param name="destination">The destination address to copy to.</param>
-        <param name="source">The source address to copy from.</param>
+        <param name="destination">The unmanaged pointer corresponding to the destination address to copy to.</param>
+        <param name="source">The unmanaged pointer corresponding to the source address to copy from.</param>
         <param name="byteCount">The number of bytes to copy.</param>
         <summary>Copies bytes from the source address to the destination address.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This API corresponds to the `cpblk` opcode.
+
+Both `destination` and `source` are assumed to be aligned to the machine's natural word size. See ECMA-335, Sec. III.3.30 ("cpblk - copy data from memory to memory") for more information.
```

Comment:
"natural word size" is somewhat confusing/ambiguous even in relation to the spec reference.

Some other wording that clarifies this is just saying the alignment must be `sizeof(nint)` (the same alignment as required for a pointer) would be immensely better.